.devcontainer/devcontainer.json

```diff
@@ -52,5 +56,5 @@
       ]
     }
   },
-  "postCreateCommand": "npm ci && npx gulp installDependencies"
+  "postCreateCommand": "npm i && npx gulp installDependencies && npm i -g gulp"
```

Comment:
Why change from `ci` to `i`? With `ci` it already has the dependency information in the lockfile.

Reply:
I was having an issue with jest not getting installed correctly - but I don't think this was the cause.  I will revert